demo/starter/handout-cover.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
+    <div
+      data-handout-page
+      class="break-after-page h-[270mm] px-16 py-20 flex flex-col justify-between text-slate-900 bg-gradient-to-br from-slate-50 via-white to-slate-100"
+    >
+      <div>
+        <span class="tracking-[0.35em] text-xs font-semibold uppercase text-sky-600">
+          Slidev Starter Template
+        </span>
+        <h1 class="mt-6 text-6xl font-bold border-b-4 border-slate-900 inline-block pr-4">
+          Welcome to Slidev
+        </h1>
+        <p class="mt-6 max-w-xl text-lg leading-relaxed text-slate-700">
+          A companion handout for the demo deck that showcases Slidev's text-first authoring, navigation helpers, and Vue-driven
+          enhancements. Explore the pages ahead to follow along without missing any of the live interactions.
```

Comment:
As this is the starter, I wanted to keep it simple. Could you reduce the content of `handing-cover` and `handout-ending` to only keep the minimal content for demonstration.